### PR TITLE
User crud UN-140

### PR DESCRIPTION
**Pull request by PThorpe92**
Tests are included for the controller 

TODO: add pagination and sorting for Index, look into middleware

### CODE REVIEW

File: app/Http/Controllers/v1/UserController.php
Comment (chrissantillan) [inline app/Http/Controllers/v1/UserController.php:1]: @PThorpe92 should this file be in `app/Http/Controllers/**Api**/v1`?

Comment (PThorpe92) [reply app/Http/Controllers/v1/UserController.php:1]: Yeah this is something we need to figure out:

The route itself would actually be `api/v1` either way if you define it in the `routes/api.php`

If you look into the controller, you'll see it has pre-generated methods (commented out) that return views, this confused me, and kind of seems to blend the two. I guess we need to determine if we are going to separate everything.

Comment (chrissantillan) [reply app/Http/Controllers/v1/UserController.php:1]: Yeah. Was just wondering if we were keeping the same directory structure from Starfish in v2, where API controllers are kept in `Api/V1` and regular controllers (?) are kept in the parent Controllers dir.

**Review by chrissantillan — APPROVED** [review]
Approved. Going forward, team will need to agree upon directory structure for Controllers (front end view-based and back end API based)

**Review by nokierae — APPROVED** [review]

**Pull request merged**